content/en/blog/2024/kubecon-china-2024.md

```diff
@@ -0,0 +1,56 @@
+---
+title:
+  OpenTelemetry Talks at Kubecon + CloudNativeCon + Open Source Summit + AI_dev
+  China 2024
+linkTitle: Kubecon China 2024
+date: 2024-06-25
+# prettier-ignore
+cSpell:ignore: Alhamdani Censhare EBPF Hrabusa Husni Huxing Jiahang Krom Sianturi Wanqi Zhang Zihao Ziyi
+author: '[Tiffany Hrabusa](https://github.com/tiffany76)'
+draft: true # TODO: remove this line once your post is ready to be published
+sig: Communications SIG
+---
+
+Join the OpenTelemetry project maintainers and members of the governance and
+technical committees at the [KubeCon + CloudNativeCon + Open Source Summit
++ AI_dev] conference in Hong Kong from August 21 to 23, 2024.
```

Comment:
```suggestion
Join the OpenTelemetry project maintainers at the [KubeCon + CloudNativeCon + Open Source Summit + AI_dev][] conference in Hong Kong from August 21 to 23, 2024.
```

As far as I know no member of the committees is attending the conference in person, cc @open-telemetry/governance-committee @open-telemetry/technical-committee 

Also, the link is broken in the preview

Reply:
Fixed link and removed mention of committee members in 0f5542e.